run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$NODE_ENV" = "ci" ]
+then echo "starting service"
+  SITEROOT=/firearms GA_TAG_ID=UA-72527484-1
```

Comment:
Sadly bad commit messages do not explain it: https://github.com/UKHomeOffice/gro/commit/069eca1f